### DISCUSSION TITLE
建议为3.0版本增加备份/升级功能

### DISCUSSION THREAD

**Discussion by ghost** [AlistGo/alist#1647] [Ideas]
AList开发组成员们您们好！
感谢您们为我们开发的AList，ta灰常方便了我的生活。
请问能否为Windows版本的AList增加一个像是V2版本的备份功能和（或）升级功能呢，这样能直接方便升级了。

**Comment by cxw620** [comment]
我也奇怪为什么砍掉这个功能了

**Reply by ghost** [reply]
我也提了Issue，没人理我...

**Comment by ghost** [comment]
https://github.com/alist-org/alist/issues/1642
low priority

**Comment by itsHenry35** [comment]
同求，这样迁移起来方便一些

**Comment by leoneoo** [comment]
同求，这样迁移起来方便一些

**Comment by TommyMyCat** [comment]
同求，谢谢

**Comment by goodrabbit** [comment]
同求，这个功能有必要且也实用。

**Comment by E5-lab** [comment]
同求，这样多电脑配置一次就可以直接应用了

**Comment by kevzcr** [comment]
原来不是我一个人有这样的想法和要求，大家都。。。

**Comment by msycn** [comment]
把opt/alist/data.db备份就行了

**Reply by ghost** [reply]
我知道，但是：
1.我用的是Windows系统，我希望能有像v2那样子的JSON文件
2.我懒，一键升级不香吗？像这种万一忘了怎么办？如果有个一键升级和（或）一键导数据不就更方便了吗？

**Reply by msycn** [reply]
开源软件,自己fork增加就行

**Reply by ghost** [reply]
我哪会啊，如果大家都会，就没人在这里提issue了。
这种软件是为大家带来便利的，不是为大家带来学习压力的！
行了，到此为止吧。
